docs/core-concepts/core-sdk/cli/known-issues.md

```diff
@@ -0,0 +1,67 @@
+# Known Issues for the .NET Core CLI Tools
+
+This document contains the known issues and suggested workarounds for 
+the Command Line Interface (CLI) tools in the .NET Core SDK Preview 1 release.
+
+A known issue is a major issue that block users in doing their everyday tasks and that affect all or 
+most of the commands in the CLI tools. If you want to report an issue, you can submit one at the [CLI repo Issues](https://github.com/dotnet/cli/issues) page. 
+
+## OpenSSL dependency on OS X
+
+OS X "El Capitan" (10.11) comes with version 0.9.8 of OpenSSL. .NET Core depends on an OpenSSL version equal or larger than 1.0.1. You can update the OpenSSL version by using [Homebrew](https://brew.sh), [MacPorts](https://www.macports.org/) or manually. The important bit is that you need to have the required OpenSSL version on the path when you work with .NET Core. 
+
+**Workaround**
+
+With Homebrew, you can run the following commands to get this done: 
+
+```console
+brew update
+brew install openssl
+brew link --force openssl
+```
+
+MacPorts doesn't have the concept of linking, so it is recommended that you uninstall 0.9.8 version of OpenSSL using the following command:
+
+```console
+sudo port upgrade openssl
+sudo port -f uninstall openssl @0.9.8
+```
+
+You can verify whether you have the right version using the `openssl version` command from the Terminal. 
+
+## Users of zsh (z shell) don't get dotnet on the path after install
+There is a known issue in the `oh-my-zsh` installer that interferes with how `path_helper` works on OS X systems. In short, the said installer creates a `.zshrc` file which contains the exploded path at the time of installation.
+This clobbers any dynamically generated path, such as the one generated by `path_helper`. 
+
+There is an [outstanding PR](https://github.com/robbyrussell/oh-my-zsh/pull/4925) on the `oh-my-zsh` repo for this. 
+
+**Workaround 1**
+
+Symlink the `dotnet` binary in the installation directory to a place in the global path, such as, `/usr/local/bin`.
+You can use the following command to create the symbolic link:
+
+```console
+ln -s /usr/local/share/dotnet/dotnet /usr/local/bin
+```
+
+**Workaround 2**
+
+Edit your `.zshrc` and/or `.zshprofile` files to add the `/usr/local/share/dotnet` directory to the $PATH. 
+
+## dotnet restore times out on Windows 7 64-bit edition
+
+**Symptom**
+
+If you have VirtualBox and you try to use the CLI tools on a machine with Windows 7 64-bit edition installed, the `dotnet restore` command will be really slow and will eventually time out without restoring your project.
```

Comment:
I need to fix this. It is not just Virtual Box, it is pretty much any virtual networking interface.